src/environments/environment.prod.ts

```diff
@@ -1,7 +1,7 @@
 export const environment = {
   production: true,
   hostname: 'identity.bitclout.com',
-  nodeHostname: 'bitclout.com',
+  nodeHostname: 'localhost:18001',
```

Comment:
Same comment as on the frontend PR.  Is this correct?  Seems weird to have localhost as an endpoint in the prod.ts file

Reply:
just for testing purposes - reverted.